ptx/sec_deriv_intro.ptx

```diff
@@ -546,7 +593,14 @@
           <caption>A graph of <m>f(x)=3x^2+5x-7</m>, along with its normal line at <m>x=1</m></caption>
           <!-- START figures/fig_normal1.tex -->
           <image xml:id="img_normal1">
-            <description></description>
+            <shortdescription>
+              A graph of function 3x^2+5x-7 along with its normal line at x=1.
+            </shortdescription>
+            <description>
+              The graph starts at the origin. Both x and y axes increase by <m>1</m>, the x-axis ends at <m>4</m> in the diagram while the y-axis ends at <m>3</m>.
```

Comment:
`<m>x</m> and <m>y</m> axes`...  x-axis should be `<m>x</m> axis`, y-axis should be `<m>y</m> axis`